sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -2319,9 +2315,14 @@ CurlConnection::CurlConnection(
     }
   }
 
-#if defined(AZ_PLATFORM_LINUX)
   if (!options.CAPath.empty())
   {
+#if defined(AZ_PLATFORM_LINUX)
```

Comment:
This should just rely on the curl error being returned.

Reply:
Agreed. I plan to remove this ifdef and if curl correctly handles an unsupported CA Path passed in correctly.